### ISSUE TITLE
Setup repo, reOCR and test for completeness for propositioner and utskottsbetänkanden

### ISSUE THREAD

**Issue by MansMeg** [welfare-state-analytics/riksdagen-corpus#380]
Pelle had a good point that we might want to find missing propositions and utskottsbetänkande early on so there is time to complete these documents if there are documents missing.

**Comment by fredrik1984** [comment]
Good point. When @Lottabrorsson and our colleagues at the library have found the "missing" motions, then they can do a similar review for propositions and committee reports.
